packages/core-kit/src/language.ts

```diff
@@ -1,7 +1,12 @@
+import { LanguageTag, languageTagGuard } from '@logto/language-kit';
 import { z } from 'zod';
 
-import { fallback } from './utilities/zod';
+import { fallback } from '@/utilities';
 
+export const getDefaultLanguageTag = (language: string): LanguageTag =>
+  languageTagGuard.or(fallback<LanguageTag>('en')).parse(language);
+
+// TODO: remove old language related type, constants and utility function below after merging Localization feature into master
```

Comment:
Maybe add an issue and paste the issue number here.

Reply:
Have added LOG-4250